compose/src/main/kotlin/com/yourssu/handy/compose/Chip.kt

```diff
@@ -0,0 +1,104 @@
+package com.yourssu.handy.compose
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.yourssu.handy.compose.foundation.HandyTextStyle
+import com.yourssu.handy.compose.foundation.HandyTypography
+
+sealed class ChipSize(
+    val height: Dp,
+    val iconSize: IconSize = IconSize.XS,
+    val horizontalPadding: Dp = 12.dp,
+    val round: Dp = 40.dp
+) {
+    data object Small : ChipSize(height = 24.dp)
+    data object Medium : ChipSize(height = 32.dp)
+}
+
+/**
+ * 정보를 입력하거나 선택하여 컨텐츠를 필터링할 수 있는 [Chip]입니다.
+ *
+ * @param onCheckedChange Chip의 선택 상태가 변경될 때 호출되는 함수
+ * @param text Chip 내부 텍스트
+ * @param sizeType Chip 사이즈 설정 : Small, Medium(Default)
+ * @param leftIcon Chip 내부 텍스트의 왼쪽에 표시되는 아이콘
+ * @param rightIcon Chip 내부 텍스트의 오른쪽에 표시되는 아이콘
+ * @param checked Chip의 선택 여부
+ * @param enabled Chip의 활성화 여부
+ */
+@Composable
+fun Chip(
+    onCheckedChange: (Boolean) -> Unit,
+    modifier: Modifier = Modifier,
+    text: String,
```

Comment:
기본값이 없는 파라미터(필수 파라미터)의 경우 modifier보다 위에 두는 것을 권장합니다! 기본값이 있는 optional 파라미터는 modifier 밑으로 두면됩니다

```suggestion
    onCheckedChange: (Boolean) -> Unit,
    text: String,
    modifier: Modifier = Modifier,
```
[참고](https://cheonjaeung.com/posts/jetpack-compose-modifier-rules/)